### PR TITLE
- Update.

### PR DESCRIPTION
**Pull request by etiennenoel**

### CODE REVIEW

File: packages/aws/src/clients/cloudformation.client.ts [hunk 1]
```diff
@@ -60,18 +60,18 @@ export class CloudformationClient implements CloudformationClientInterface {
...
+    async getStackDescription(stackName: string): Promise<Stack | undefined> {
         this.logHandler.debug("CLOUDFORMATION CLIENT - Getting stack information", {stackName}, AwsModuleKeyname);
         const command = new DescribeStacksCommand({
             StackName: stackName,
         })
         try {
             const response: DescribeStacksCommandOutput = await this.getClient().send(command);
             if(!response.Stacks || response.Stacks.length < 1){
-                throw new Error("No stacks were returned from cloudformation");
+                return undefined;
```
Comment (mathieugh) [inline packages/aws/src/clients/cloudformation.client.ts:71]: I think maybe we should log a warning here as well ?

File: packages/aws/src/clients/cloudformation.client.ts [hunk 2]
```diff
@@ -80,6 +80,21 @@ export class CloudformationClient implements CloudformationClientInterface {
         }
     }
 
+    /**
+     * Gets the description and all its details from all the CloudFormation stacks.
+     */
+    async listStacks(): Promise<Stack[] | undefined> {
```
Comment (mathieugh) [inline packages/aws/src/clients/cloudformation.client.ts:86]: Can this command really return undefined or just an empty array ?

**Review by mathieugh — CHANGES_REQUESTED** [review]

File: packages/aws/src/clients/cloudformation.client.ts (hunk 2 above)
Comment (etiennenoel) [reply packages/aws/src/clients/cloudformation.client.ts:86]: Apparently undefined but you are right I prefer an empty array.

File: packages/aws/src/clients/cloudformation.client.ts (hunk 1 above)
Comment (etiennenoel) [reply packages/aws/src/clients/cloudformation.client.ts:71]: No because it's the only way to know if a stack already exists, it's to query it.

**Pull request merged**